src/components/MediaPlayer/MediaPlayer.md

```diff
@@ -5,19 +5,13 @@ MediaPlayer component provides a player that facilitates both audio and video me
 - `enableFileDownload` : accepts a Boolean value, which has a default value of `false` and is not required. Once this is set to `true` it adds an icon to the player's toolbar to display `rendering` files in the Canvas and enables downloading them. This is a custom feature added to the VideoJS instance in Ramp.
 - `enablePIP` : accepts a Boolean value, which has a default value of `false` and is not required. When this is set to `true`, it adds an icon to the player's toolbar to enable Picture-In-Picture feature for the current player. This icon is a VideoJS feature.
 
+A sample usage of this component is displayed in the `IIIFPlayer` component.
 
-To import this component from the librayr;
+To import and use this component from the library;
 ```js static
-import { MediaPlayer } from '@samvera/ramp';
-```
-
-```jsx padded
-import IIIFPlayer from '../IIIFPlayer/IIIFPlayer';
-import config from '../../../env.js';
+import { IIIFPlayer, MediaPlayer } from '@samvera/ramp';
 
-<IIIFPlayer
-  manifestUrl={`${config.url}/manifests/${config.env}/lunchroom_manners.json`}
->
+<IIIFPlayer manifestUrl={manifest_url} manifest={manifest}>
```

Comment:
What should `manifestUrl` and `manifest` be?  Should you only use one at a time or both?
I think it would be good to standardize the documented usage and/or clarify what each prop is and why you'd use one or the other or both.
FWIW This looks the same as `StructuredNavigation.md`, but `IIIFPlayer.md` and `MetadataDisplay.md` only use `manifest` and explicitly set it to `lunchroom_manners.js`.

Reply:
In the `IIIFPlayer.md` the props are explained. It mentions that the `manifest` prop takes precedence over the `manifestUrl`.

And in the `MediaPlayer.md` and `StructuredNavigation.md` I removed the actual code because it replicate the components displayed at top with `IIIFPlayer` component. Let me change it so it's not confusing and looks the same as the others. Thanks for spotting this :100: